arch/inst/Zclsd/c.sdsp.yaml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: c.sdsp
+long_name: Stack-pointer based store doubleword from even/odd register pair
+description: |
+  Stores a stack-pointer relative 64-bit value from registers rs2' and rs2'+1. It computes an effective
+  address by adding the zero-extended offset, scaled by 8, to the stack pointer, x2. It expands to sd
+  rs2, offset(x2).
```

Comment:
To match the "assembly", below:
- change `rs2'` to `rs2`
- change `x2` to `sp`
- put backquotes around `sd rs2, offset(sp)`